modules.yml

```diff
@@ -1065,3 +1065,13 @@
   path: reporting/xml
   required_keys: []
   version: '1.1'
+- author: Ryan Hays (@_ryanhays)
+  dependencies: []
+  description: Imports hosts and ports into the respective databases from Masscan
+    XML output.
+  files: []
+  last_updated: '2020-04-07'
+  name: Masscan XML Output Importer
+  path: import/masscan
+  required_keys: []
+  version: '1.0'
```

Comment:
Files should end with a line break. Either that's PEP8, or I am OCD like that and have just always done it that way.